acct-group/headscale/headscale-0.ebuild

```diff
@@ -0,0 +1,8 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit acct-group
+
+KEYWORDS="~amd64"
```

Comment:
uhh don't add keywords to acct-* ebuilds, it's already set in the [eclasses](https://gitweb.gentoo.org/repo/gentoo.git/tree/eclass/acct-user.eclass#n144)